main/webapp/modules/core/styles/views/data-table-view.css

```diff
@@ -7,13 +9,13 @@ Redistribution and use in source and binary forms, with or without
 modification, are permitted provided that the following conditions are
 met:
 
-    * Redistributions of source code must retain the above copyright
+* Redistributions of source code must retain the above copyright
```

Comment:
I think all changes to this file are spurious, so you could revert the file to its previous version.